### PR TITLE
Bug 1710701 - Cannot scroll through comments on TikTok videos

### PR DESCRIPTION
**Pull request by wisniewskit**

### CODE REVIEW

File: src/injections/css/bug1710701-tiktok.com-comments-scrollbar.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * www.tiktok.com - Ensure that the comment section for videos has a vertical
+ *                  scrollbar when needed.
+ */
```
Comment (ksy36) [inline src/injections/css/bug1710701-tiktok.com-comments-scrollbar.css:4]: Could you please add the link to webcompat and bugzilla for the reference?

**Review by ksy36 — APPROVED** [review]
LGTM!

File: src/injections/css/bug1710701-tiktok.com-comments-scrollbar.css (hunk 1 above)
Comment (wisniewskit) [reply src/injections/css/bug1710701-tiktok.com-comments-scrollbar.css:4]: Yes, odd that I forgot it :)

**Comment by ksy36** [comment]
looks like the linter is failing at `src/data/injections.js` ?

**Comment by wisniewskit** [comment]
Yeah, that's annoying. I just fixed the issue, so we can merge this tomorrow once the checks go green.

**Pull request merged**